lib/src/components/_stacks-tags.less

```diff
@@ -0,0 +1,120 @@
+//
+//  TAGS
+//
+//  TABLE OF CONTENTS
+//  • BASE STYLE
+//  • STYLE VARIATIONS
+//  • LAYOUTS & SIZES
+//
+//  ===========================================================================
+//  $   MIXIN
+//      Any tag we use will always modify the same items, so to save time,
+//      here's a mixin to state all that CSS.
+//  ---------------------------------------------------------------------------
+& { // create a single scope to load the config into, at least for now
+#stacks-internals #load-config();
+
+.tag-styles(@border: transparent, @background: transparent, @color: inherit) {
+    border-color: @border;
+    background-color: @background;
+    color: @color;
+
+    .s-tag--dismiss {
+        background-color: transparent;
+        color: fade(@color, 90%);
+    }
+
+    &:hover {
+        border-color: darken(@border, 5%);
+        background-color: darken(@background, 5%);
+        color: darken(@color, 5%);
+
+        .s-tag--dismiss {
+            background-color: fade(@color, 10%);
+            color: darken(fade(@color, 90%), 5%);
+
+            &:hover {
+                background-color: fade(@color, 90%);
+                color: @white;
+            }
+        }
+    }
+}
+
+
+//  ===========================================================================
+//  $   BASE STYLE
+//      The same style is applied to all tags. Do not modify the core style.
+//  ---------------------------------------------------------------------------
+.s-tag {
+    display: inline-flex;
+    align-content: center;
+    justify-content: center;
+    min-width: 0;
+    padding: 0 (@su4 + @su2);
+    border-style: solid;
+    border-width: 1px;
+    border-radius: @tags-border-radius;
```

Comment:
I don't think we'll actually need to customize this. I think it can just be `@br-sm`.